develop/components/pages/TooltipPage.tsx

```diff
@@ -30,33 +89,108 @@ const TooltipPage: React.FunctionComponent = () => {
                     <p>Here are sample outputs</p>
                     <div className="result">
                         <Tooltip
-                            message="Tooltip message could be long, therefore, controlling the position and width is important"
-                            position="right"
-                            width={200}
-                        />
-                    </div>
-
-                    <p>Custom svg and can be dismissed when clicked outside</p>
-                    <div className="result">
-                        <Tooltip
-                            message="Tooltip message could be long, therefore, controlling the position and width is important"
-                            position="right"
-                            width={200}
-                            customSvg={moneySVG}
-                            id="dismissable-tooltip"
+                            content={content === 0 && (hasNodeAsContent ? nodeTooltipContent : defaultTooltipContent)}
```

Comment:
its a bit confusing to have content as a number with values 1,2,3 its not clear what they mean, maybe you want to create an enum instead,

Reply:
added enum